packages/yew-macro/tests/html_macro/html-if-pass.rs

```diff
@@ -0,0 +1,17 @@
+use yew::prelude::*;
+
+fn compile_pass() {
+    html! { if true {} };
+    html! { if true { <div/> } };
+    html! { if true { <div/><div/> } };
```

Comment:
```suggestion
    html! { if true { <div/><div/> } };
    html! { if true { <><div/><div/></> } };
```
Not sure if it is worth adding this case.
I can see lots of users using fragments in the if/else blocks so this check could catch if we disallow fragments in the future (for whatever reason) that we are making a breaking change.